_data/simple-icons.json

```
@@ -14441,8 +14441,8 @@
         },
         {
             "title": "SoundCloud",
-            "hex": "FF3300",
-            "source": "https://soundcloud.com/press"
+            "hex": "FF5500",
+            "source": "https://press.soundcloud.com/media_kits/228462/"
```

The logos available at this URL are PNG/JPG - you can get away with the below URL if you're getting the logo from the header:

```suggestion
            "source": "https://press.soundcloud.com"
```